sia/wallet/transactions.go

```diff
@@ -186,12 +183,12 @@ func (w *Wallet) SignTransaction(id string, wholeTransaction bool) (transaction
 	defer w.mu.Unlock()
 
 	// Fetch the transaction.
-	ot, exists := w.transactions[id]
+	openTransaction, exists := w.transactions[id]
 	if !exists {
 		err = errors.New("no transaction found for given id")
 		return
 	}
-	transaction = *ot.transaction
+	transaction = *openTransaction.transaction
```

Comment:
You don't want to pass around pointers to transactions, because if someone modifies the transaction you'll be in trouble.

Also this just adheres to the convention that you should only use pointers if it's aboslutely necessary, and never as an optimization.